virtcontainers/kata_agent.go

```diff
@@ -1073,7 +1073,7 @@ func (k *kataAgent) handleBlockVolumes(c *Container) []*grpc.Storage {
 		}
 		if c.sandbox.config.HypervisorConfig.BlockDeviceDriver == VirtioBlock {
 			vol.Driver = kataBlkDevType
-			vol.Source = blockDrive.VirtPath
+			vol.Source = blockDrive.PCIAddr
```

Comment:
Is there any way we can add a unit test for this?

Reply:
It seems there is no unit test for func handleBlockVolumes, I can add one later.